src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -38,6 +38,9 @@ const SignedChatScreen = () => {
 
   const {moveToAnonymousChannel} = useMoveChatTypeHook();
 
+  const exitedGroup = selectedChannel?.rawJson?.channel?.better_channel_member?.findIndex(
+    (item: any) => item.user_id === signedProfileId
+  );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `findIndex` method is used to check if the user has exited the group. However, it's not clear what value `better_channel_member` holds when a user exits a group. If it's `null` or `undefined`, this line could throw an error. Consider adding optional chaining (`?.`) before `findIndex` to prevent potential errors.
```diff
- const exitedGroup = selectedChannel?.rawJson?.channel?.better_channel_member?.findIndex(
+ const exitedGroup = selectedChannel?.rawJson?.channel?.better_channel_member?.findIndex(
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->